### PR TITLE
PEOPLE: Data quality in TableDisplay

### LINKED ISSUE #792: PEOPLE: Datenqualität als Spalte in Personenliste
based on #791 

Als Mitgliederverwaltung SAC will ich in einer Personenliste sehen, welche Personen Datenqualitätsissues haben.

Neu soll eine optionale Spalte 'Datenqualität' in den Personenlisten einblendbar sein. Diese basiert auf dem Wert `Person#data_quality` und wird als entsprechendes Symbol angezeigt (siehe ev. auch Login Spalte):

- error: [times-circle](https://fontawesome.com/v5/icons/times-circle?s=solid)
- warning: [exclamation-triangle](https://fontawesome.com/v5/icons/exclamation-triangle?f=classic&s=solid)
- info: [info-circle](https://fontawesome.com/v5/icons/info-circle?f=classic&s=solid)
- ok: [check-circle](https://fontawesome.com/v5/icons/check-circle?f=classic&s=solid)

Auf dem Icon erscheint ein Tooltip mit dem entsprechenden Namen: Fehler, Warnung, Info, OK

Die Spalte soll sortierbar sein, wobei die Reihenfolge error, warning, info, ok; bzw umgekehrt ist.

# ToDo

- [x] View/Controller anpassen
- [x] Specs schreiben
- [x] Mit angemessener Rolle "durchklicken"

### PR DESCRIPTION
**Pull request by hunchr**
Fixes #792

image

### CODE REVIEW

File: app/domain/table_displays/resolving_column.rb [hunk 1]
```diff
@@ -27,8 +27,8 @@ def render(attr)
       end
     end
 
-    def sort_by(_attr)
-      nil
+    def sort_by(attr)
+      'people.data_quality' if attr == :data_quality
     end
```
Comment (hunchr) [inline app/domain/table_displays/resolving_column.rb:32]: es wird zwar sortiert, aber nicht richtig

File: spec/features/people/list_spec.rb [hunk 2]
```diff
@@ -67,4 +67,12 @@
...
     end
   end
+
+  it "allows showing the data_quality column" do
+    visit group_people_path(group_id: group.id)
+    click_link("Spalten")
+    check("Datenqualität")
+    click_link("Spalten")
+    expect(page).to have_css('td[data-attribute-name="data_quality"]')
+  end
```
Comment (njaeggi) [inline spec/features/people/list_spec.rb:77]: I would suggest adding a spec for all different icons possible and then checking

`expect(find("td[data-attribute-name="data_quality"] i")).to have_css('fas fa-check-circle')`
or
`expect(find("td[data-attribute-name="data_quality"] i")).to have_css('fas fa-check-info')`

**Review by njaeggi — CHANGES_REQUESTED** [review]
LGTM, just some small suggestions

**Review by njaeggi — APPROVED** [review]
:rocket:

**Pull request merged**